src/pip/_internal/locations.py

```diff
@@ -9,6 +9,8 @@
 import sysconfig
 from distutils import sysconfig as distutils_sysconfig
 from distutils.command.install import SCHEME_KEYS  # type: ignore
+from distutils.command.install import install
+from typing import cast
```

Comment:
pip can't have a run-time dependency on typing; hence the "MYPY_CHECK_RUNNING" code block.